### PR TITLE
added: user info to the pending and getinfo api

### PR DESCRIPTION
**Pull request by nevil-mathew**

### CODE REVIEW

File: src/services/connections.js [hunk 1]
```diff
@@ -82,6 +88,31 @@ module.exports = class ConnectionHelper {
 				})
 			}
 
+			const defaultOrgId = await getDefaultOrgId()
```
Comment (aks30) [inline src/services/connections.js:91]: @nevil-mathew we can skip making an API call here if we just need default org id

**Review by aks30 — CHANGES_REQUESTED** [review]
Reviewed on 15 Oct

File: src/services/connections.js (hunk 1 above)
Comment (nevil-mathew) [reply src/services/connections.js:91]: Updated `getDefaultOrgId()` to return from the environment if it exists.

File: src/services/connections.js [hunk 2]
```diff
@@ -96,10 +127,47 @@ module.exports = class ConnectionHelper {
...
+				})
+			}
+			const friendIds = connections.rows.map((connection) => connection.friend_id)
+			let friendDetails = await menteeQueries.getUsersByUserIds(friendIds, {
+				attributes: ['user_id', 'name', 'designation', 'organization_id'],
+			})
+			const userExtensionsModelName = await menteeQueries.getModelName()
+
+			const uniqueOrgIds = [...new Set(friendDetails.map((obj) => obj.organization_id))]
+			friendDetails = await entityTypeService.processEntityTypesToAddValueLabels(
```
[the remark this answers is not in the document]
Comment (nevil-mathew) [reply src/services/connections.js:148]: FE need the role to be shown in the message request tab.

**Pull request merged**